### ISSUE TITLE
Why is the official site redirected to GitHub page?

### ISSUE THREAD

**Issue by zhuxinghan** [winjs/winjs#1683]
Though the project is not maintained, you should leave the official site try.buildwinjs.com for developers who still use WinJS. We still need the docs on the site.

**Comment by AngelMunoz** [comment]
@zhuxinghan I find these docs winjs.azurewebsites.net/ to be more complete and a little bit more extensive 
also these may come handy for you https://docs.microsoft.com/en-us/previous-versions/windows/apps/mt502392(v%3dwin.10)

**Comment by zhuxinghan** [comment]
@AngelMunoz Thank you very much. Hope those docs won't be deleted again.

**Comment by AngelMunoz** [comment]
Yeah I completely Agree also there should be wording declaring this obsolete/unmaintained
